sh.fhs.KatawaShoujoReEngineered.yaml

```diff
@@ -0,0 +1,108 @@
+app-id: sh.fhs.KatawaShoujoReEngineered
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: "ksre"
+add-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    add-ld-path: "."
+    version: "22.08"
+    autodelete: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --filesystem=~/.renpy
```

Comment:
this should be "persist" instead.

Reply:
this is path to filesystem, not inside sandbox — to read and write to common save location for the game, in case someone will install it after playing downloaded version, or want to import saves

Reply:
no. it should just persist. opening a sandbox hole for that isn't desirable.

Reply:
okay, closing the sandbox